packages/schemas/tables/sign_in_experiences.sql

```diff
@@ -9,6 +10,7 @@ create table sign_in_experiences (
   language_info jsonb /* @use LanguageInfo */ not null,
   terms_of_use_url varchar(2048),
   privacy_policy_url varchar(2048),
+  agree_to_terms_policy agree_to_terms_policy not null default 'AutoAgree',
```

Comment:
can you add some comments thus the generated type can show them?

```suggestion
  /** Some single line comments */
  agree_to_terms_policy agree_to_terms_policy not null default 'AutoAgree',
```